src/components/radio-list/radio-list.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./radio-list.scss";
+
+/**
+ * RadioList component - input radio list
+ * @param {string} label
+ * @param {string} description
+ * @param {[{ key: string, value: string, checked?:boolean }]} options List of select options with key, value and selected properties
+ * @param {'vertical'|'horizontal'} type List orientation: 'vertical', 'horizontal'
+ */
+export const RadioList = ({ label, description, options, onChange, type }) => {
+  const classNames = ["field", "radio-list"];
+  if (type === "horizontal") {
+    classNames.push("horizontal");
+  }
+  if (type === "vertical") {
+    classNames.push("vertical");
+  }
+  return (
+    <div className={classNames.join(" ")}>
+      <label className="label">{label}</label>
+      <p className="subtitle is-2">{description}</p>
+      <div className="control">
+        {options &&
+          options.map(option => {
```

Comment:
I generally like seeing the parameter destructured so `option.` isn't repeated in the return value. (Also, implicit return here). Not really a functional feedback though.